proto/viam/component/camera/v1/camera.proto

```diff
@@ -12,29 +12,47 @@ option java_package = "com.viam.component.camera.v1";
 
 // A CameraService services all cameras associated with a robot
 service CameraService {
-  // GetImage returns a frame from a camera of the underlying robot. A specific MIME type
-  // can be requested but may not necessarily be the same one returned.
+  // GetImage returns a frame from a camera of the underlying robot. A specific
+  // MIME type can be requested but may not necessarily be the same one
+  // returned.
   rpc GetImage(GetImageRequest) returns (GetImageResponse) {
+    option deprecated = true;
     option (google.api.http) = {get: "/viam/api/v1/component/camera/{name}/image"};
   }
 
   rpc GetImages(GetImagesRequest) returns (GetImagesResponse) {
+    option deprecated = true;
     option (google.api.http) = {get: "/viam/api/v1/component/camera/{name}/images"};
   }
 
-  // RenderFrame renders a frame from a camera of the underlying robot to an HTTP response. A specific MIME type
-  // can be requested but may not necessarily be the same one returned.
+  // RenderFrame renders a frame from a camera of the underlying robot to an
+  // HTTP response. A specific MIME type can be requested but may not
+  // necessarily be the same one returned.
   rpc RenderFrame(RenderFrameRequest) returns (google.api.HttpBody) {
+    option deprecated = true;
     option (google.api.http) = {get: "/viam/api/v1/component/camera/{name}/render_frame"};
   }
 
-  // GetPointCloud returns a point cloud from a camera of the underlying robot. A specific MIME type
-  // can be requested but may not necessarily be the same one returned.
+  // GetPointCloud returns a point cloud from a camera of the underlying robot.
+  // A specific MIME type can be requested but may not necessarily be the same
+  // one returned.
   rpc GetPointCloud(GetPointCloudRequest) returns (GetPointCloudResponse) {
+    option deprecated = true;
     option (google.api.http) = {get: "/viam/api/v1/component/camera/{name}/point_cloud"};
```

Comment:
We wouldn't deprecate these APIs immediately as part of this proposal, but rather, we would remove them at the end of the project once all important camera models have been migrated to the new API that can express all of them.